cg/meta/workflow/analysis.py

```diff
@@ -26,6 +28,8 @@ def __init__(self, pipeline: Pipeline, config: Optional[dict] = None):
 
     @property
     def threshold_reads(self):
+        """Defines whether workflow case should be automatically analyzed
+        even when not all samples have adequate read count"""
```

Comment:
this documentation of the functionality makes me a bit confused: "even when not all samples have adequate read count".
Doesn't this functionality define what the "adequate read count" is?
suggestion?
"Defines the threshold for adequate read count for whether workflow case should be automatically analyzed"

Reply:
What defines adequate read count is application tag.
This property does not define threshold, only that it should or should not be applied! The threshold is set in status_db in application table

Reply:
Updated